frontend/rust-lib/flowy-document2/tests/parser/html_text/test.rs

```diff
@@ -0,0 +1,107 @@
+use crate::parser::html_text::utils::{match_html, match_text};
+
+#[tokio::test]
+async fn heading_test() {
+  let source = include_str!("../../assets/json/heading.json");
+  let expect_html = include_str!("../../assets/html/heading.html");
+  match_html(source, expect_html);
+  let expect_text = include_str!("../../assets/text/heading.txt");
+  match_text(source, expect_text);
+}
+
+#[tokio::test]
+async fn callout_test() {
```

Comment:
how about using a for loop to combine these tests

Reply:
Let me think about it because there are a few methods with some differences